extensions/amp-a4a/0.1/test/test-amp-a4a.js

```diff
@@ -314,6 +316,22 @@ describe('amp-a4a', () => {
       });
     });
 
+    it('populate postAdResponseExperimentFeatures', () => {
```

Comment:
Nit: "it populate" doesn't flow well, so "it populates" would be preferable. But most other names don't flow with the "it" prefix either.

Reply:
Done